db/schemas/manifestSchema20160105.json

```diff
@@ -0,0 +1,367 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id":"https://e-manifest.18f.gov/schemas/manifestSchema20160105.json",
+    "title": "e-Manifest JSON Schema",
+    "description": "First draft of JSON Schema.  I fully expect this to be editted and revised.  Resource used to make this was http://spacetelescope.github.io/understanding-json-schema/index.html",
+
+    "definitions":{
+        "address": {
+             "type": "object",
+            "properties":{
+                "address_1":   {
+                    "description": "Address line 1",
+                    "type": "string",
+                    "maxLength":30 ,
+                    "minLength":0 
+                },
+                "address_2":   {
+                    "description": "Address line 2",
+                    "type": "string",
+                    "maxLength":30 ,
+                    "minLength":0 
+                },
+                "city":   {
+                    "description": "City name",
+                    "type": "string",
+                    "maxLength":25 ,
+                    "minLength":0 
+                },
+                "state":   {
+                    "description": "Two character abbreviation for State or Province",
+                    "type": "string",
+                    "maxLength":2,
+                    "minLength":2, 
+                    "COMMENT":" add Enummerated value"
+                },
+                "zip_code":   {
+                    "description": "Postal Code",
+                    "type": "string",
+                    "maxLength":14,
+                    "minLength":0, 
+                    "COMMENT":" add Regular Expression"
+                }
+            }
+        },
+
+        "TelephoneNumberDataType":  {
```

Comment:
I've always used snake case when creating JSON APIs with Ruby because that is a Ruby convention. 

I recently read, however, that it is preferred to use snake case since JSON is JavaScript and js convention is to use CamelCase. 

I am fine with either as long as we are consistent.